apidef/oas/oas.go

```diff
@@ -277,27 +277,27 @@ func (s *OAS) getTykOperations() (operations Operations) {
 }
 
 // AddServers adds a server into the servers definition if not already present.
-func (s *OAS) AddServers(apiURL string) {
-	if len(s.Servers) == 0 {
-		s.Servers = openapi3.Servers{
-			{
-				URL: apiURL,
-			},
-		}
-		return
+func (s *OAS) AddServers(apiURLs ...string) {
```

Comment:
This is, technically, a breaking change - if we're doing that, could you please add an error return, and make sure to return an error when len(apiURLs) == 0 - handle the error outside of here.

On a side-note, do we need it to be public? Can it be `addServers` and left unexposed?